<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="mohammeds-MBP.fios-router.home" ignored="0" name="Add Customer Test" tests="2" failures="2" timestamp="04 Feb 2019 08:41:29 GMT" time="0.796" errors="0">
  <testcase name="addCustomerTest" time="0.0" classname="com.w2a.testcases.AddCustomerTest">
    <failure type="java.lang.AssertionError" message="Failing Test Case">
      <![CDATA[java.lang.AssertionError: Failing Test Case
	at com.w2a.testcases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:28)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- addCustomerTest -->
  <testcase name="addCustomerTest" time="0.0" classname="com.w2a.testcases.AddCustomerTest">
    <failure type="java.lang.AssertionError" message="Failing Test Case">
      <![CDATA[java.lang.AssertionError: Failing Test Case
	at com.w2a.testcases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:28)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- addCustomerTest -->
</testsuite> <!-- Add Customer Test -->
